tests/network/test_decoder.py

```diff
@@ -14,13 +16,29 @@ def setUp(self) -> None:
         self.output_size = 9
 
     def test_whenInstantiateASeq2SeqModel_thenParametersAreOk(self):
-        decoder = Decoder(self.input_size_dim, self.hidden_size, self.num_layers, self.output_size)
+        decoder = Decoder(self.input_size_dim,
+                          self.hidden_size,
+                          self.num_layers,
+                          self.output_size,
+                          attention_mechanism=False)
 
         self.assertEqual(self.input_size_dim, decoder.lstm.input_size)
         self.assertEqual(self.hidden_size, decoder.lstm.hidden_size)
         self.assertEqual(self.num_layers, decoder.lstm.num_layers)
         self.assertEqual(self.output_size, decoder.linear.out_features)
 
+    def test_whenInstantiateASeq2SeqAttModel_thenParametersAreOk(self):
```

Comment:
The name of the methods should be different & reflect the difference in the test cases

Reply:
There is but the difference is hard to read. The Att before model.